topics/variant-analysis/tutorials/trio-analysis/tutorial.md

```diff
@@ -0,0 +1,441 @@
+> > - The pattern here is `^[0-9MYX]`, which can again be broken down into different components
+> >      - `^` means that the pattern has to start at the beginning of the line not somewhere randomly in the line. This way our prefix `chr` will be inserted at the start of the line.
+> >      - `[0-9MYX]` means that at this position there should be a character from the list `[]`, namely either a number from `0-9` or the character `M`, `Y`, or `X`.
+> > 
+> {: .comment}
+>
+{: .hands_on}
+
+## Normalizing VCF
+After decompressing and adding the prefixes to the VCFs we need to normalize the variants in the VCF to standardize how the variants are represented within the VCF. This is a very important step, since the variants in the mother and daughter might be represented differently, which would mean that the causative variant might be overlooked!
+
+One of the normalization steps is splitting multiallelic variants, 2 variants detected on the same position but on a different allele. Splitting these records will put the 2 variants on a separate line, that way the impact of the individual mutations can be evaluated. In addition, indels need to be left-aligned and normalized because that’s how they are stored in the annotation databases. An indel is left-aligned and normalized, according to {% cite Tan2015 %}, "if and only if it is no longer possible to shift its position to the left while keeping the length of all its alleles constant" and "if it is represented in as few nucleotides as possible".
+
+| ![plot](../../images/trio-analysis/variant_normalization.png) |
+|:--:|
+| **Figure 2:** *The different ways to represent INDELs [(source)](https://academic.oup.com/bioinformatics/article/31/13/2202/196142).* |
+
+> ### {% icon hands_on %} Hands-on: Normalize VCF
+>
+> 1. {% tool [bcftools norm](toolshed.g2.bx.psu.edu/repos/iuc/bcftools_norm/bcftools_norm/1.9+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"VCF/BCF Data"*: `VCFs with chr prefix` (output of **Text reformatting** {% icon tool %})
+>    - *"Choose the source for the reference genome"*: `Use a built-in genome`
+>        - *"Reference genome"*: `Human (Homo sapiens): hg19`
+>    - *"Left-align and normalize indels?"*: `Yes`
+>    - *"~multiallelics"*: `split multiallelic sites into biallelic records (-)`
+>    - In *"Restrict all operations to"*:
+>        - *"Regions"*: `Do not restrict to Regions`
+>        - *"Targets"*: `Do not restrict to Targets`
+>    - *"output_type"*: `uncompressed VCF`
+>
+> > ### {% icon tip %} Tip: Check the normalization summary
+> >
+> > You can have a look at the summary to check what changes were made. First, expand the output of bcftools norm (by clicking on the box) and it should be listed in the box. If not you can find it by clicking on the icon view details {% icon details %} and look at the output of the ToolStandard Error.
+> >
+> {: .tip}
+>
+{: .hands_on}
+
+## Filter NON_REF sites

```

Comment:
I have decided to use the _Filter data on any column using simple expressions_ tool. I would like to replace the NON_REF sites before loading the VCFs since that step takes more than 1 day if I keep the NON_REF sites. e61e6713377e9e1a815ea31629471c7ec0d4cf2e